src/components/missions/index.jsx

```diff
@@ -1,76 +1,53 @@
-            </td>
-            <td className="td">
-              <span className="status-not-a-member">NOT A MEMBER</span>
-            </td>
-            <td className="td">
-              <button className="join-mission-btn" type="button">
-                Join Mission
-              </button>
-            </td>
-          </tr>
-          <tr>
-            <td className="td">Thaicom</td>
-            <td className="td">
-              Lorem ipsum dolor, sit amet consectetur adipisicing elit. Molestiae esse consectetur
-              inventore quod possimus veniam ipsum ab quam recusandae ad eius minus in nulla
-              veritatis, dolorem doloremque harum sed non, itaque nesciunt? Dolorum dolore
-              asperiores debitis! Exercitationem, optio esse quasi nisi in, obcaecati fuga eum
-              cumque mollitia aliquid culpa officia.
-            </td>
-            <td className="td">
-              <span className="status-not-a-member">NOT A MEMBER</span>
-            </td>
-            <td className="td">
-              <button className="join-mission-btn" type="button">
-                Join Mission
-              </button>
-            </td>
-          </tr>
-          <tr>
-            <td className="td">Thaicom</td>
-            <td className="td">
-              Lorem ipsum dolor, sit amet consectetur adipisicing elit. Molestiae esse consectetur
-              inventore quod possimus veniam ipsum ab quam recusandae ad eius minus in nulla
-              veritatis, dolorem doloremque harum sed non, itaque nesciunt? Dolorum dolore
-              asperiores debitis! Exercitationem, optio esse quasi nisi in, obcaecati fuga eum
-              cumque mollitia aliquid culpa officia.
-            </td>
-            <td className="td">
-              <span className="status-not-a-member">NOT A MEMBER</span>
-            </td>
-            <td className="td">
-              <button className="join-mission-btn" type="button">
-                Join Mission
-              </button>
-            </td>
-          </tr>
-        </tbody>
-      </table>
-    </div>
-  </section>
-);
+const Missions = () => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getMissions());
+  }, [dispatch]);
+
+  const missions = useSelector((state) => state.missionsReducer.missions);
+  const missionsList = missions.map(
+    ({ description, mission_id: missionId, mission_name: missionName }) => (
+      <tr key={missionId}>
+        <td className="td title-container">
+          <h1 className="mission-title">{missionName}</h1>
+        </td>
+        <td className="td">{description}</td>
+        <td className="td td-with-span">
+          <span className="status-not-a-member">NOT A MEMBER</span>
+        </td>
+        <td className="td button-td">
+          <button className="join-mission-btn" type="button">
+            Join Mission
+          </button>
+        </td>
+      </tr>
```

Comment:
[Optional] 
Try to make this `tr` reusable component to make it more scalable in the future